openshift/operator-controller/kustomize/overlays/openshift/olmv1-ns/metrics/operator_controller_metrics_monitor_servicemonitor.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: metrics-monitor
+  labels:
+    control-plane: operator-controller-controller-manager
+    openshift.io/cluster-monitoring: 'true'
+spec:
+  endpoints:
+    - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+      interval: 30s
+      path: /metrics
+      port: https
```

Comment:
What is this value numerically? Does it match with the operator-controller metrics port (8443)?

Reply:
Note that the op-con metrics port is different than catalogd.

Reply:
yes, see https://github.com/openshift/operator-framework-operator-controller/pull/252#discussion_r1944945062